src/app/home/home.component.scss

```diff
@@ -151,3 +151,10 @@ h4 {
 .info {
   height: 300px;
 }*/
+
+.button-row > div > a {
+  width: 90% !important;
```

Comment:
I'm always wary of !important, although I guess it's scoped to just this component, so maybe it doesn't matter.

Reply:
Ya I figured because it is contained to the scope of the component it would not be as big of a deal. I'll try and reduce the usage of these.